cdap-ui/app/features/overview/controllers/overview-ctrl.js

```diff
@@ -46,7 +47,15 @@ function ($scope, $state, myLocalStorage, MY_CONFIG, Widget, MyMetricsQueryHelpe
     if (serviceStatuses.indexOf('NOTOK') === -1) {
       this.systemStatus = 'green';
     }
-  }.bind(this));
+  }.bind(this),
+  function error(res) {
+    if (res.auth_uri) {
+      EventPipe.emit('backendDown', 'Invalid Auth Token. Please refresh to re-authenticate.');
```

Comment:
If Auth Token is invalid, do we want to disable the entire UI or just display a message? We can reroute to the login screen and display an error message right?